<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>2188</width>
    <height>1450</height>
   </rect>
  </property>
  <property name="focusPolicy">
   <enum>Qt::ClickFocus</enum>
  </property>
  <property name="windowTitle">
   <string>薪资统计</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>冲浪.png</normaloff>冲浪.png</iconset>
  </property>
  <property name="windowOpacity">
   <double>1.000000000000000</double>
  </property>
  <property name="styleSheet">
   <string notr="true">background-image: url(:/新前缀/未标题-2.png);</string>
  </property>
  <widget class="QSlider" name="strength">
   <property name="geometry">
    <rect>
     <x>1172</x>
     <y>1182</y>
     <width>915</width>
     <height>22</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::StrongFocus</enum>
   </property>
   <property name="toolTip">
    <string>调整力度</string>
   </property>
   <property name="maximum">
    <number>1000</number>
   </property>
   <property name="singleStep">
    <number>10</number>
   </property>
   <property name="value">
    <number>500</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="invertedAppearance">
    <bool>false</bool>
   </property>
   <property name="tickPosition">
    <enum>QSlider::TicksBelow</enum>
   </property>
   <property name="tickInterval">
    <number>100</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_strength">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>910</y>
     <width>181</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Comic Sans MS</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(25, 76, 113);</string>
   </property>
   <property name="text">
    <string>Strength</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="mypos">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>1050</y>
     <width>101</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>华光细圆_CNKI</family>
     <pointsize>9</pointsize>
     <kerning>true</kerning>
    </font>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;我的位置&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="origin">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>1050</y>
     <width>81</width>
     <height>18</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="font">
    <font>
     <family>华光细圆_CNKI</family>
     <pointsize>9</pointsize>
     <kerning>true</kerning>
    </font>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;原点&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="left">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>1050</y>
     <width>81</width>
     <height>18</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="font">
    <font>
     <family>华光细圆_CNKI</family>
     <pointsize>9</pointsize>
     <kerning>true</kerning>
    </font>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;左框点&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="right">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>1050</y>
     <width>81</width>
     <height>18</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="font">
    <font>
     <family>华光细圆_CNKI</family>
     <pointsize>9</pointsize>
     <kerning>true</kerning>
    </font>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;右框点&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>1070</y>
     <width>731</width>
     <height>28</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QPushButton" name="MyPosbutton">
      <property name="font">
       <font>
        <family>Arial</family>
       </font>
      </property>
      <property name="focusPolicy">
       <enum>Qt::StrongFocus</enum>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(25, 76, 113);
color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>MyPos</string>
      </property>
      <property name="shortcut">
       <string>Z</string>
      </property>
      <property name="autoDefault">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="EnemyPosbutton">
      <property name="font">
       <font>
        <family>Arial</family>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(25, 76, 113);
color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>EnemyPos</string>
      </property>
      <property name="shortcut">
       <string>X</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="Originbutton">
      <property name="font">
       <font>
        <family>Arial</family>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(25, 76, 113);
color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>Origin</string>
      </property>
      <property name="shortcut">
       <string>`</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="Leftbutton">
      <property name="font">
       <font>
        <family>Arial</family>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(25, 76, 113);
color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>Left</string>
      </property>
      <property name="shortcut">
       <string>[</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="Rightbutton">
      <property name="font">
       <font>
        <family>Arial</family>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(25, 76, 113);
color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>Right</string>
      </property>
      <property name="shortcut">
       <string>]</string>
      </property>
     </widget>
    </item>
   </layout>
   <zorder>Originbutton</zorder>
   <zorder>MyPosbutton</zorder>
   <zorder>Leftbutton</zorder>
   <zorder>Rightbutton</zorder>
   <zorder>EnemyPosbutton</zorder>
  </widget>
  <widget class="QLabel" name="enemypos">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>1050</y>
     <width>101</width>
     <height>20</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="font">
    <font>
     <family>华光细圆_CNKI</family>
     <pointsize>9</pointsize>
     <kerning>true</kerning>
    </font>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;敌人位置&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="distancelabel">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>940</y>
     <width>191</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Yu Gothic UI Semilight</family>
     <pointsize>12</pointsize>
     <bold>true</bold>
     <stylestrategy>PreferAntialias</stylestrategy>
     <kerning>true</kerning>
    </font>
   </property>
   <property name="layoutDirection">
    <enum>Qt::RightToLeft</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(25, 76, 113)
</string>
   </property>
   <property name="text">
    <string>水平屏距：</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
   <property name="wordWrap">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QCheckBox" name="opencurve">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>920</y>
     <width>171</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>华光粗圆_CNKI</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
color: rgb(0, 0, 0);
</string>
   </property>
   <property name="text">
    <string>抛物线模式开/关</string>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QCheckBox" name="modifycurve">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>960</y>
     <width>211</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>华光粗圆_CNKI</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
color: rgb(0, 0, 0);
</string>
   </property>
   <property name="text">
    <string>停止自动识别角度</string>
   </property>
   <property name="shortcut">
    <string>C</string>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>1130</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>华光粗圆_CNKI</family>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(28, 87, 129);</string>
   </property>
   <property name="text">
    <string>透明度调整</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QSlider" name="transaction">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>1130</y>
     <width>101</width>
     <height>22</height>
    </rect>
   </property>
   <property name="maximum">
    <number>100</number>
   </property>
   <property name="value">
    <number>100</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>20</y>
     <width>801</width>
     <height>871</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgba(129, 176, 210,60);
</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>0</y>
     <width>801</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>楷体</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="contextMenuPolicy">
    <enum>Qt::DefaultContextMenu</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(28, 87, 129);
color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>菜鸟快乐挂</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>890</y>
     <width>821</width>
     <height>291</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
     <kerning>true</kerning>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>rect.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="pushbutton_close">
   <property name="geometry">
    <rect>
     <x>800</x>
     <y>10</y>
     <width>20</width>
     <height>20</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>关闭</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>10</y>
     <width>20</width>
     <height>20</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>没用</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushbutton_mini">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>10</y>
     <width>20</width>
     <height>20</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>最小化</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="anglelabel">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>1110</y>
     <width>161</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Comic Sans MS</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(121, 121, 121);</string>
   </property>
   <property name="text">
    <string>Tilit: 0</string>
   </property>
  </widget>
  <widget class="QLabel" name="zoomlabel">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>1110</y>
     <width>131</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Comic Sans MS</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(121, 121, 121);</string>
   </property>
   <property name="text">
    <string>Zoom: 1</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_angle">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>970</y>
     <width>171</width>
     <height>41</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="font">
    <font>
     <family>Comic Sans MS</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(25, 76, 113);</string>
   </property>
   <property name="text">
    <string>Angle</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QSpinBox" name="angle">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>1130</y>
     <width>41</width>
     <height>19</height>
    </rect>
   </property>
   <property name="mouseTracking">
    <bool>false</bool>
   </property>
   <property name="focusPolicy">
    <enum>Qt::WheelFocus</enum>
   </property>
   <property name="acceptDrops">
    <bool>false</bool>
   </property>
   <property name="inputMethodHints">
    <set>Qt::ImhFormattedNumbersOnly</set>
   </property>
   <property name="maximum">
    <number>120</number>
   </property>
   <property name="value">
    <number>50</number>
   </property>
  </widget>
  <zorder>label_4</zorder>
  <zorder>label_3</zorder>
  <zorder>strength</zorder>
  <zorder>label_strength</zorder>
  <zorder>mypos</zorder>
  <zorder>origin</zorder>
  <zorder>left</zorder>
  <zorder>right</zorder>
  <zorder>layoutWidget</zorder>
  <zorder>enemypos</zorder>
  <zorder>distancelabel</zorder>
  <zorder>opencurve</zorder>
  <zorder>modifycurve</zorder>
  <zorder>label</zorder>
  <zorder>transaction</zorder>
  <zorder>label_2</zorder>
  <zorder>pushbutton_close</zorder>
  <zorder>pushButton_2</zorder>
  <zorder>pushbutton_mini</zorder>
  <zorder>anglelabel</zorder>
  <zorder>zoomlabel</zorder>
  <zorder>label_angle</zorder>
  <zorder>angle</zorder>
 </widget>
 <resources>
  <include location="BgTest.qrc"/>
  <include location="BgTest.qrc"/>
 </resources>
 <connections/>
</ui>
